agent/internal/client/interface.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/evergreen-ci/evergreen/model"
 	"github.com/evergreen-ci/evergreen/model/artifact"
 	"github.com/evergreen-ci/evergreen/model/manifest"
+	"github.com/evergreen-ci/evergreen/model/patch"
```

Comment:
it looks like this package is getting imported twice now: once as patch and once as patchmodel (on line 14)